warehouse/templates/manage/project/release.html

```diff
@@ -192,8 +192,13 @@ <h3>{% trans %}Delete release{% endtrans %}</h3>
       <p>{% trans %}You will not be able to re-upload a new distribution of the same type with the same version number.{% endtrans %}</p>
       <p>{% trans %}Deletion will break any downstream projects relying on a pinned version of this package. It is intended as a last resort to address legal issues or remove harmful releases.{% endtrans %}</p>
       <p>{% trans yank_href=request.help_url(_anchor='yanked'), post_href='https://www.python.org/dev/peps/pep-0440/#post-releases', title=gettext('External link') %}Consider <a href="{{ yank_href }}" title="{{ title }}" target="_blank" rel="noopener">yanking</a> this release, making a new release or a <a href="{{ post_href }}" title="{{ title }}" target="_blank" rel="noopener">post release</a> instead.{% endtrans %}</p>
+      <ul class="no-bullets">
```

Comment:
Suggestions for the acknowledgements here to make them relevant to the action and consistent with the project delete confirmations:

- [ ] I understand that I am permanently deleting all distributions for the {{ release.version }} release of this project.
- [ ] I understand that my users will no longer be able to install this release of this project.
- [ ] I understand that I will <a href="https://pypi.org/help/#file-name-reuse">not be able to re-upload any deleted distribution files using the same file names.</a>.
- [ ] I understand that I will not be able to undo this.
- [ ] I understand that the PyPI administrators will not be able to undo this.

Reply:
Those sound great! I've added the additional ones in [c6f1259](https://github.com/pypi/warehouse/pull/13930/commits/c6f1259936646d1e03b0f454e66345a5f7705695)